back-end/src/main/java/dota/buff/service/impl/PlayerServiceImpl.java

```diff
@@ -25,17 +27,20 @@
 
     @Override
     public MatchDTO getLastMatch(long steamId) {
+        log.info("Getting last user match with steamId: {}", steamId);
         return getMatches(steamId, 1).get(0);
     }
 
     @Override
     public List<MatchDTO> getMatches(long steamId, int matches) throws DotaSparkerException {
+        log.info("Getting " + matches + " user matches with steamId: {}", steamId);
```

Comment:
You can write log.info("Getting {} user matches with steamId: {}", matches, steamId);